.pipelines/templates/build-package-for-nuget.yml

```diff
@@ -215,7 +214,8 @@ stages:
         versionSpec: 6.2.1
 
     - task: PowerShell@2
-      displayName: 'Updating NuSpec'
+      displayName: 'Updating NativeNuget.NuSpec'
+      condition: eq(${{ parameters.IsDummyPackage }}, false)
```

Comment:
consider using [conditional step insertion](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#conditionally-run-a-step).

e.g.,
```yaml
steps:
# ...
- ${{ if eq(parameters.IsDummyPackage, false) }}:
  - task: # non dummy package step...
- ${{ else }}:
  - task: # dummy package step...
```

I think the condition is more obvious here instead of in the task's `condition` value. also, the disabled step won't be displayed in the UI.